base/essentials.jl

```diff
@@ -281,6 +283,25 @@ getindex(v::SimpleVector, I::AbstractArray) = Core.svec(Any[ v[i] for i in I ]..
 
 Tests whether the given array has a value associated with index `i`. Returns `false`
 if the index is out of bounds, or has an undefined reference.
+
+```jldoctest
```

Comment:
do we want to use `rand` here? I guess it doesn't really matter because you never look at the values but as a style thing...